app/static/js/events/session/new.js

```diff
@@ -18,3 +18,55 @@ $(document).ready(function () {
         $(this).parent().parent().parent().remove();
     });
 });
+
+var remove_url = function (name) {
+    $('input[name="' + name + '_url"]').val("");
+    $('#download-' + name).hide();
+    $('#remove-' + name).hide();
+    $('#progress-' + name).hide();
+};
+
+var fileUpload = function(name, media){
+    var fd = new FormData();
+    fd.append(name, media.files[0]);
+    filename = media.files[0].name;
+    $.ajax({
+        xhr: function(){
+            var xhr = new window.XMLHttpRequest();
```

Comment:
I don't understand why there is a mixed usage of `jquery` and pure `javascript` over here. Why has this approach been taken to do this ?